scripts/cross/Dockerfile

```diff
@@ -12,12 +12,16 @@ RUN apt-get update -y && apt-get install --no-install-recommends -y -q \
                          libltdl-dev \
                          libltdl7
 
-RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
-RUN apt-get install -y nodejs npm
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
+RUN apt-get install -y nodejs
```

Comment:
Can you move this above and add it to the single apt-get command?

Reply:
I put the script and source list in one block and then put node and yarn in a single apt-get, still have two of them because we need to install curl before doing this script and the source list. After talking to Josh this morning though, I think this will be temporary as it sounds like the long-term plan is a separate Dockerfile for the JS stuff.